### PR TITLE
[docs] Update README for running e2e locally

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
cc @aravindhp @akhil-rane

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,25 @@ export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export CLUSTER_ADDR=<cluster_name, eg: ravig211.devcluster.openshift.com>
 export KUBE_SSH_KEY_PATH=<path to ssh key>
 ```
-Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
+- Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
 is being used in KUBE_SSH_KEY_PATH. Please note that we're using libra as keypair
 for our CI purposes.
-
+- Make sure that `/payload` directory exists and has tree structure mentioned like this(Since this structure can change, please make sure this is upto date)
```
Comment (aravindhp) [inline README.md]: `Ensure that /payload directory exists and is accessible by the user account. The directory needs to be populated with the following files. Please see the Dockefile for figuring where to download and build these binaries. It is up to the user to keep these files up to date.`

Please link to the Dockerfile in the sentence above.

File: README.md [hunk 2]
```diff
@@ -21,10 +21,25 @@ export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
...
+│   └── win-overlay.exe
+├── hybrid-overlay.exe
+├── kube-node
+│   ├── kubelet.exe
+│   └── kube-proxy.exe
+├── powershell
+│   └── wget-ignore-cert.ps1
+└── wmcb.exe
+```
 Once the above variables are set:
```
Comment (aravindhp) [inline README.md]: `Once the above variables are set and the /payload directory has been populated, run the following script:`

File: README.md [hunk 3]
```diff
@@ -21,11 +21,26 @@ export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export CLUSTER_ADDR=<cluster_name, eg: ravig211.devcluster.openshift.com>
 export KUBE_SSH_KEY_PATH=<path to ssh key>
 ```
-Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
+- Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
 is being used in KUBE_SSH_KEY_PATH. Please note that we're using libra as keypair
 for our CI purposes.
-
-Once the above variables are set:
+- Ensure that /payload directory exists and is accessible by the user account. The directory needs to be populated with the following files. Please see the [Dockefile](https://github.com/openshift/windows-machine-config-operator/blob/master/build/Dockerfile) for figuring where to download and build these binaries. It is up to the user to keep these files up to date.
```
Comment (ravisantoshgudimetla) [inline README.md]: Note that the tree structure for `/payload` in dockerfile, dockerfile.ci is not update in master branch

File: README.md [hunk 4]
```diff
@@ -21,11 +21,26 @@ export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export CLUSTER_ADDR=<cluster_name, eg: ravig211.devcluster.openshift.com>
 export KUBE_SSH_KEY_PATH=<path to ssh key>
 ```
-Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
+- Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what
```
Comment (aravindhp) [inline README.md]: The link should point to master.

File: README.md (hunk 3 above)
Comment (sebsoto) [inline README.md]: Dockefile -> Dockerfile

File: README.md (hunk 4 above)
Comment (aravindhp) [reply README.md]: I take that back as the position of that code could change..

File: README.md (hunk 3 above)
Comment (ravisantoshgudimetla) [reply README.md]: Done

**Comment by ravisantoshgudimetla** [comment]
> /hold cancel

Why didn't this work? I just did edited my comment.

**Pull request merged**